### DISCUSSION TITLE
[Feature - API] Delete person

### DISCUSSION THREAD

**Discussion by aviv926** [immich-app/immich#6559] [Feature Request]
### The feature

It would be absolutely nice to allow the deletion of a person from the face recognition through the API or through the browser interface, which would cause ML to create new face templates for the images where the deleted person is missing.

This can solve some problems as explained in [Discord](https://discord.com/channels/979116623879368755/1192853757499813898)

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#19511
